release/pypi/prep_binary_for_pypi.py

```diff
@@ -17,7 +18,41 @@
 from pathlib import Path
 
 
-def process_wheel(whl_file, output_dir=None):
+cuda_metadata_inject = """Requires-Dist: nvidia-cuda-nvrtc-cu12==12.6.77; platform_system == "Linux" and platform_machine == "x86_64"
```

Comment:
I am not really fan of this section. Extracting this info from https://github.com/pytorch/pytorch/blob/main/.github/scripts/generate_binary_build_matrix.py#L58 is possible However its missing triton information.